src/main/client/src/components/@commons/FilePicker.tsx

```diff
@@ -0,0 +1,127 @@
+// from https://github.com/PedroDBFlores/chakra-ui-file-picker
```

Comment:
It was a nice idea copying the code rather than importing as a package, we can modify for our use.